documentation/Changelog.md

```diff
@@ -2,10 +2,13 @@
 
 ## MSBuild 16.11.0
 
-This version of MSBuild will ship with Visual Studio 2019 version 16.11.0 and .NET SDK 5.0.400.
+This version of MSBuild shipped with Visual Studio 2019 version 16.11.0 and .NET SDK 5.0.400.
```

Comment:
NIT: I think it should be `is shipped` or for better active instead of passive, should be rephrased using `ships` as:
`This version of MSBuild ships with Visual Studio 2019 version 16.11.0 and .NET SDK 5.0.400.`

Reply:
I prefer past tense here because this release is in some ways just a pointer to the "standard" ways to get MSBuild: through VS or the .NET SDK.

Reply:
@rainersigwald 
thanks for your reply! ok, I understand now 🙂